src/coffee/sync/utils/product.coffee

```diff
@@ -562,8 +562,11 @@ class ProductUtils extends BaseUtils
               return
             deltaValue = @getDeltaValue(value)
             unless deltaValue
-              deltaValue = value[0]
-              delete deltaValue.value
+              # Taken from https://github.com/commercetools/nodejs/blob/ab8edfb41bdc7d429f20554d3d8a45ef251228f8/packages/sync-actions/src/product-actions.js#L286
```

Comment:
I just wanted to say that I did not create the code by myself, but got it from a trustworthy resource :) And in case you'll find an error here in the future, you should also check the link.